ethereum/simplestack/simple_stack.go

```diff
@@ -0,0 +1,167 @@
+package simplestack
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/0xProject/0x-mesh/ethereum/miniheader"
+)
+
+type updateType int
```

Comment:
As a general rule, if a variable is exported, the type of that variable should also be exported.

Otherwise, it can add annoying restrictions on what you can do as the caller/user of this package. For example you could not write a function outside of this package that accepts `updateType` as an argument (sometimes comes up in helper functions for tests). 

```suggestion
type UpdateType int
```